universal-application-tool-0.0.1/conf/evolutions/default/12.sql

```diff
@@ -0,0 +1,9 @@
+# --- Add versioning and lifecycle management.
+
+# --- !Ups
+
+alter table questions add repeater_id jsonb;
+
```

Comment:
Suggestion (blocking): please give a default value for existing questions so as not to break our staging environment.

You want something like
```sql
update questions set repeater_id = '<default value in json>' where repeater_id is null;
```

Reply:
Now the intended default is null. Should I do anything for that?

Reply:
Nope, adding the nullable column will default to the value to null for all existing rows

Reply:
Yup. In this case null is expected value and won't break staging.